### DISCUSSION TITLE
Where to download tar file for USB booting?

### DISCUSSION THREAD

**Discussion by JetpackJackson** [Maccraft123/Cadmium#47] [Q&A] [answered]
In the instructions where it says to "download and uncompress cadmium-<device>.tar.gz to your pendrive," I don't see any place to download any files. I could just be missing it completely... if I am, could someone point me in the right direction? Thanks in advance!

**Comment by apple-corps** [comment, marked as answer]
In releases:

https://github.com/Maccraft123/Cadmium/releases

**Reply by JetpackJackson** [reply]
Oh, thanks! I completely missed that! My device doesn't seem to be supported though :(

**Reply by apple-corps** [reply]
Not sure what ARM device you have. But a good place to discuss might be https://discord.gg/ZZbwyvKCmV
